addons/website_sale_comparison/views/website_sale_comparison_view.xml

```diff
@@ -18,11 +18,19 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">product.attribute.category</field>
         <field name="view_mode">tree</field>
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+                Create a new attribute category
```

Comment:
This is actually missing the .pot, missed that during the review, could you create a new PR to add it?

Reply:
Thanks for spotting this (PR added [here](https://github.com/odoo/odoo/pull/107974)) :+1: